realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncedRealmMigrationTests.java

```diff
@@ -107,7 +108,7 @@ public void missingFields_hiddenSilently() {
                 .addField("newField", String.class);
         // A schema version has to be set otherwise Object Store will try to initialize the schema again and reach an
         // error branch. That is not a real case.
-        dynamicRealm.setVersion(0);
+        OsObjectStore.setSchemaVersion(dynamicRealm.getSharedRealm(), 0);
```

Comment:
Why not have this implementation inside `dynamicRealm.setVersion(0)`, that seems more readable when using the `dynamicRealm`. With this change new developers need to know they have to go here to change the version number.